assets/models/system/Mistral-7B-Instruct-v0-2-generic-gpu/spec.yaml

```diff
@@ -0,0 +1,24 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: mistralai-Mistral-7B-Instruct-v0-2-generic-gpu
+version: 1
+path: ./
+tags:
+  foundryLocal: ""
+  license: "apache-2.0"
+  licenseDescription: "This model is provided under the License Terms available at <https://www.apache.org/licenses/LICENSE-2.0.html>."
+  author: Microsoft
+  inputModalities: "text"
+  outputModalities: "text"
+  task: chat-completion
+  maxOutputTokens: 2048
+type: custom_model
+variantInfo:
+  parents:
+  - assetId: azureml://registries/azureml/models/mistralai-Mistral-7B-Instruct-v0-2/versions/1
```

Comment:
Will this work given that parent model already existed (i.e., wasn't created here)? Does version info need to be added for parent?